src/commands/cmd_key.cc

```diff
@@ -222,7 +221,7 @@ class CommandExpireAt : public Commander {
 
     timestamp_ = *parse_result;
 
-    return Commander::Parse(args);
+    return Status::OK();
```

Comment:
Why do we change this line?

Reply:
Just to unify. Because the other functions in this file are written like this.

Reply:
ok, that makes sense since it does nothing inside the parent's Parse().